README.rst

```diff
@@ -70,44 +70,32 @@ Please note that on Linux installing voice you must install the following packag
 * libffi-dev (or ``libffi-devel`` on some systems)
 * python-dev (e.g. ``python3.6-dev`` for Python 3.6)
 
-Quick Example
---------------
-
-.. code:: py
-
-    import nextcord
-
-    class MyClient(nextcord.Client):
-        async def on_ready(self):
-            print('Logged on as', self.user)
-
-        async def on_message(self, message):
-            # don't respond to ourselves
-            if message.author == self.user:
-                return
-
-            if message.content == 'ping':
-                await message.channel.send('pong')
-
-    client = MyClient()
-    client.run('token')
 
 Bot Example
 ~~~~~~~~~~~~~
 
 .. code:: py
 
-    import nextcord
     from nextcord.ext import commands
 
-    bot = commands.Bot(command_prefix='>')
+    class Bot(commands.Bot):
+        def __init__(self, *args, **kwargs):
+            super().__init__(*args, **kwargs)
+
+        async def on_ready(self):
+            print(f'Logged in as {self.user} (ID: {self.user.id})')
+            print('------')
+
+
+    bot = Bot(command_prefix=commands.when_mentioned_or('$'))
 
     @bot.command()
-    async def ping(ctx):
-        await ctx.send('pong')
+    async def hello(ctx: commands.Context):
+        await ctx.reply('Hello!')
```

Comment:
.send should be used here

Reply:
I don't think so. The original was [this](https://github.com/Rapptz/discord.py/blob/master/examples/reply.py) and it specifically replied to the user.

Reply:
My bad.